src/config/dark-sites.config

```diff
@@ -73,20 +73,24 @@ blackhatworld.com
 blackle.com
 blacklemag.com
 blackmod.net
+blastpremier.com
 blink.sh
 blizzard.com
 block.wingysam.xyz
 blog.aractus.com
+blog.counter-strike.net
 blog.dota2.com
 blog.ja-ke.tech
 blox.link
+blueprintjs.com
```

Comment:
https://blueprintjs.com/docs/

Reply:
They have a dark theme option, I thought it was applied by default since I tried visiting the site from different browsers.

Reply:
> They have a dark theme option, I thought it was applied by default since I tried visiting the site from different browsers.

They don't, I had `privacy.resistFingerprinting` to `true` to prefer a light-colour-scheme and will automatically prefer light colour scheme.